lxd/network/driver_ovn.go

```diff
@@ -4252,17 +4252,15 @@ func (n *ovn) InstanceDevicePortIPs(instanceUUID string, deviceName string) ([]n
 
 // InstanceDevicePortStop deletes an instance device port from the internal logical switch.
 func (n *ovn) InstanceDevicePortStop(ovsExternalOVNPort openvswitch.OVNSwitchPort, opts *OVNInstanceNICStopOpts) error {
```

Comment:
Worth removing this func too if it just returns nil? Maybe it can be even removed from the `ovnNet` interface?

Reply:
Nvm, it's removed in the next commit.
Is it worth not having the change to `return nil` in this commit?

Reply:
ah just saw this now, yeah potentially, although it showed my logical steps towards the final result.